Move External Identities APIs to v1.0 (#12210)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/assignmentorder.md

---
title: "assignmentOrder resource type"

> description: "Used to define the order of the attributes being collected within a user flow."

Reviewer: ```suggestion
description: "Defines the order of the attributes being collected within a user flow."
```

author: "jkdouglas"
localization_priority: Normal
ms.prod: "identity-and-sign-in"
doc_type: resourcePageType
---

# assignmentOrder resource type

Namespace: microsoft.graph

> Used to define the order of the attributes being collected within a user flow. THis determines how the attribute collection page is displayed when a user signs up via a user flow.

Reviewer: ```suggestion
Used to define the order of the attributes being collected within a user flow. The order determines how the attribute collection page is displayed when a user signs up using a user flow.
```

## Properties

|Property|Type|Description|
|:---|:---|:---|

> |order|String collection|A list of identityUserFlowAttribute IDs provided to determine the order in which attributes should be collected within a user flow.|

Reviewer: ```suggestion
|order|String collection|A list of identityUserFlowAttribute object identifiers that determine the order in which attributes should be collected within a user flow.|
```
